packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -99,16 +167,19 @@ export class FauxClassGenerator {
 
     this.logFetchedObjects(standardSObjects, customSObjects);
 
-    this.generateFauxClasses(
-      standardSObjects,
-      path.join(sobjectsFolderPath, this.STANDARDOBJECTS_DIR)
-    );
-    this.generateFauxClasses(
-      customSObjects,
-      path.join(sobjectsFolderPath, this.CUSTOMOBJECTS_DIR)
-    );
+    try {
+      this.generateFauxClasses(standardSObjects, standardSObjectsFolderPath);
+    } catch (exc) {
```

Comment:
I just found out that it's probably more idiomatic to use `e` rather than `exc`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch

Reply:
will do